Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverProfileSummary.hs

```diff
@@ -84,8 +85,9 @@ getDriverProfileSummary (driverId, _, _) = do
         missedEarnings <- B.runInReplica $ BQ.findFareForCancelledBookings cancelledBookingIdsByDriver
         driverSelectedFare <- B.runInReplica $ FPQ.findDriverSelectedFareEarnings farePramIds
         customerExtraFee <- B.runInReplica $ FPQ.findCustomerExtraFees farePramIds
-        deadKmFare <- B.runInReplica $ FPPDQ.findDeadKmFareEarnings farePramIds
-        let bonusEarnings = driverSelectedFare + customerExtraFee + deadKmFare
+        deadKmFareProgressive <- B.runInReplica $ FPPDQ.findDeadKmFareEarnings farePramIds
+        deadKmFareRental <- B.runInReplica $ FPRDQ.findDeadKmFareEarnings farePramIds
```

Comment:
Please don't do this query randomly. Only do it for Rental rides only else will unnecessary take time.

Reply:
I mean only query inside `FPRDQ` if fare type in Rentals.